python/hsfs/training_dataset.py

```diff
@@ -161,62 +162,62 @@ def compute_statistics(self):
         if self.statistics_config.enabled:
             return self._statistics_engine.compute_statistics(self, self.read())
 
-    def feed(
+    def tf_data(
         self,
-        target_name,
-        split=None,
-        feature_names=None,
-        is_training=True,
-        cycle_length=2,
+        target_name: str,
+        split: Optional[str] = None,
+        feature_names: Optional[list] = None,
+        var_len_features: Optional[list] = [],
+        is_training: Optional[bool] = True,
+        cycle_length: Optional[int] = 2,
     ):
         """
-        :param target_name: name of the target variable
-        :type target_name: str, required
-        :param split: training dataset split name, train, test or eval, defaults to None
-        :type split: str, optional
-        :param feature_names: name of training variables, defaults to None
-        :type feature_names: 1d array, optional
-        :param is_training:  whether it is for training, testing or eval, defaults to True
-        :type  is_training: boolean, optional
-        :param cycle_length: number of files to be read and deserialized in parallel, defoults to 2
-        :type cycle_length: int, optional
-        :param engine: execution engine. defaults  to spark
-        :type engine: str, optional
-        :return: feed model engine object
-        :rtype: FeedModelEngine
+        tf_data function returns TFDataEngine object that has utility methods for tf.data
+
+        # Arguments
+            training_dataset: training dataset name
+            split: name of training dataset split. train, test or eval
+            target_name: name of the target variable
+            feature_names: name of training variables
+            var_len_features: feature names that have variable length and needs to be returned as `tf.io.VarLenFeature`
+            is_training: whether it is for training, testing or eval
+            cycle_length: number of files to be read and deserialized in parallel. defaults to `2`
```

Comment:
please pay more attention and be consistent.
```suggestion
            target_name: Name of the target variable.
            split: Name of training dataset split. For example, `"train"`, `"test"` or `"val"`, defaults to `None`,
                returning the full training dataset.
            feature_names: Names of training variables, defaults to `None`.
            var_len_features: Feature names that have variable length and need to be returned as `tf.io.VarLenFeature`, defaults to `[]`.
            is_training: Whether it is for training, testing or validation. Defaults to `True`.
            cycle_length: Number of files to be read and deserialized in parallel, defaults to `2`.
```